.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,31 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
+# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: macOS-latest
```

Comment:
Please include a linux build or switch to linux entirely. Thanks

Reply:
Done.